src/main/java/pages/ProductsPage.java

```diff
@@ -94,11 +101,26 @@ public String getKitInCartText() {
         return kitInCart.getText();
     }
 
+    public String getKitInCartAttribute() {
+        wait.untilVisible(kitInCart);
+        return kitInCart.getAttribute("");
```

Comment:
not sure this method will return something. You need to pass name of attribute but you are passing empty string "" and I do not know what Selenium will return

Reply:
ok in test I saw you compare it with NULL. 
it's not valid check. You ask if element has "" empty attribute, selenium can't wind that attribute and returns NULL and in test you compare it with NULL. 
take any available attribute e.g. class and check its value.
But this is also should have context. E.g. check if class has disabled word or enabled word or hidden to identify that object is dispalyed, hidden, etc